public/styles.css

```diff
@@ -729,3 +730,69 @@ td {
 	padding: 2%;
 	text-align: center;
 }
+#QATitle {
+	font-size: 30px;
+	font-weight: bold;
+	padding-bottom: 30px;
+}
+#QList{
+	padding: 100px;
+}
+#QComp, #AComp {
+	padding: 10px;
+}
+#QBody {
+	font-size: 20px;
+	font-weight: bold;
+}
+#QUser {
+	font-size: small;
+	font-weight: normal;
+	float: right;
+
+}
+#ALetter {
+	float: left;
+	font-size: 20px;
+	font-weight: bold;
+}
+#AWrapper {
+	padding: 3px;
+}
+#Abody {
+	margin-left: 30px;
+	font-size: 20px;
+}
+
+.modal {
```

Comment:
Since we all have modals and we all pretty much used the same CSS code, it may be worth renaming these classes so there is no accidental overwriting. Maybe consider adding 'q', 'a', or 'qa' at the beginning?

Reply:
Good catch, went ahead and made those changes too so their shouldn't be any conflicts in the future.